library/src/main/java/com/novoda/downloadmanager/CompletedDownloadBatch.java

```diff
@@ -1,41 +1,38 @@
 package com.novoda.downloadmanager;
 
-import java.util.Collections;
 import java.util.List;
 
-public class Migration {
+public class CompletedDownloadBatch {
```

Comment:
Only used for `Completed` downloads that we just need to move the files for and insert into the database.

Reply:
Felt better than having a generic `Migration` object where it was capturing information that may not be required for the two different types of migration.